src/components/base/Select/index.js

```diff
@@ -136,12 +127,31 @@ const IconSelected = styled(Box).attrs({
   opacity: ${p => (p.selected ? 1 : 0)};
 `
 
+const AngleDown = props => (
+  <Box color="grey" alignItems="center" justifyContent="center" {...props}>
+    <svg viewBox="0 0 16 16" width="16" height="16">
```

Comment:
same, this can be in a constant imo. OR this can be a pure component. (but it's minor ;) )

Reply:
Maybe we can do this for all icons